go/vt/vttablet/tabletserver/throttle/throttler.go

```diff
@@ -724,15 +755,42 @@ func (throttler *Throttler) Operate(ctx context.Context, wg *sync.WaitGroup) {
 			case <-mysqlCollectTicker.C:
 				if throttler.IsOpen() {
 					// frequent
+					// Always collect self metrics:
```

Comment:
Now that this PR fixes the dormant issue, we also notice that any throttler should be collecting their own already-probed metrics. This comes at nearly no-cost. dormancy was made for the PRIMARY to not overwhelm network traffic when not needed, whilst collecting metrics from the replicas.